ibm_catalog.json

```diff
@@ -1,8179 +1,3476 @@
-									"RA-5(c)",
-									"RA-5(d)",
-									"RA-5(1)",
-									"RA-5(2)",
-									"RA-5(3)",
-									"SA-3(a)",
-									"SA-3(d)",
-									"SA-8",
-									"SA-10(a)",
-									"SA-10(b)",
-									"SA-10(c)",
-									"SA-10(d)",
-									"SA-10(e)",
-									"SA-10(1)",
-									"SA-15(a)",
-									"SI-2(a)",
-									"SI-2(b)",
-									"SI-2(c)",
-									"SI-2(d)",
-									"SI-2(2)",
-									"SI-7",
-									"SI-10"
-								]
-							}
-						]
-					},
-					"architecture": {
-						"diagrams": [
-							{
-								"diagram": {
-									"url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-devsecops-alm/main/reference-architectures/diagram-deploy-arch-ibm-devsecops-alm-diagram.svg",
-									"caption": "DevSecOps Toolchains",
-									"type": "image/svg+xml",
-									"thumbnail_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-devsecops-alm/main/reference-architectures/diagram-deploy-arch-ibm-devsecops-alm-diagram.svg"
-								},
-								"description": "The DevSecOps Application Lifecycle Management deployable architecture consists of three Continuous Delivery Toolchains, one each for development (Continuous Integration), production deployment (Continuous Deployment), and ongoing compliance validation for the deployed app (Continuous Compliance). Together they enable secure and compliant application delivery on IBM Cloud."
-							}
-						],
-						"features": [
+    "products": [
+        {
+            "label": "DevSecOps Application Lifecycle Management",
+            "name": "deploy-arch-ibm-devsecops-alm",
+            "version": "2.1.12",
+            "product_kind": "solution",
+            "tags": [
+                "enterprise_app",
+                "ibm_created"
+            ],
+            "keywords": [
+                "DevSecOps Toolchain Compliance"
+            ],
+            "short_description": "DevSecOps provides a set of predefined continuous integration, continuous deployment and continuous compliance toolchain templates.",
+            "long_description": "This deployable architecture provides tools to deploy your application using DevSecOps. See the [Deployment guide](https://cloud.ibm.com/docs/devsecops-alm) before beginning. DevSecOps toolchains use Continuous Delivery (Git Repos and Issue Tracking, Tekton pipelines, DevOps insights, and Code Risk Analyzer). DevSecOps also uses Secrets Manager, Key Protect, Cloud Object Storage, Container Registry and Vulnerability Advisor. Out of the box, DevSecOps also uses popular scanning tools such as SonarQube, Gosec, OWASP Zap (dynamic scan), any unit test framework, and GPG signing. DevSecOps can also be used with other tools such as external Git providers and artefact stores. DevSecOps supports hybrid deployments, in particular it uses private pipeline workers. DevSecOps can also be interfaced with other deployment tools such as Satellite Config and ArgoCD.\n",
```

Comment:
This deployable architecture provides tools to securely deploy application code using DevSecOps pipelines. Out of the box, these pipelines use popular scanning tools such as SonarQube, Gosec, OWASP Zap (dynamic scan), any unit test framework, and GPG for image signing. Please refer to the [Deployment guide](https://cloud.ibm.com/docs/devsecops-alm?topic=devsecops-alm-devsecops-alm-planning-req) to understand the requirements beforehand.